MomsSpaghetti/Pages/Index.cshtml.cs

```diff
@@ -117,13 +116,13 @@ public async Task<IActionResult> OnGet()
 
 
                     }
-                  
+
 
                 }
                 catch (Exception err)
                 {
                     //return RedirectToPage("Error", err.Message);
-
+                    Console.WriteLine(err);
```

Comment:
Good idea, as you don't want to leave a catch block blank.  That hides important information about what went wrong.

Go one step further.  Use proper logging instead of Console.WriteLine
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine doesn't have.
- No one will see Console.WriteLine, as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine.